addons/web/static/src/views/fields/x2many/x2many_field.js

```diff
@@ -53,15 +53,10 @@ export class X2ManyField extends Component {
 
         const { activeActions, controls } = this.archInfo;
         if (this.props.viewMode === "kanban") {
-            this.controls = controls.length
-                ? controls
-                : [
-                      {
-                          type: "create",
-                          string: this.props.addLabel || _t("Add"),
-                          class: "o-kanban-button-new",
-                      },
-                  ];
+            this.controls = (controls.length && controls) || [];
+            if (this.controls.length === 0) {
+                this.kanbanAddLabel = this.props.addLabel || _t("Add %s", this.field.string);
+            }
```

Comment:
probably not necessary to store that on `this` => can be "generated" on the fly in get rendererProps